### ISSUE TITLE
DayPicker control - Month and year switching buttons are not accessible via keyboard and have no aria-labels

### ISSUE THREAD

**Issue by M-Povarau** [ant-design/ant-design#32513]
- [x]  I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/xlsfs)

### Steps to reproduce
1. Open the code sandbox link
2. Navigate to DatePicker control using Tab and hit enter which will open the calendar
3. Press Tab and arrow keys to navigate through dates 
4. Press Shift + Tab to switch to "Next Year" button

 * Expected Result
    Focus moved to "Next Year" button

 * Actual Result
    Nothing happens, focus is on the place as it was before 

### What is expected?
Next/previous month/year should be accessible via keyboard and have appropriate aria-labels. Please see an [example of accessible calendar](https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/datepicker-dialog.html) made by W3. 

Expected sequence of steps: 
1. Open the code sandbox link
2. Navigate to DatePicker control using Tab and hit enter which will open the calendar
3. **Press Tab to move to current date in calendar** 
4. Press Shift + Tab to switch to "Next Year" button

### What is actually happening?
Next/previous month/year buttons are not accessible via keyboard. 

| Environment | Info |
|---|---|
| antd | 4.16.13 |
| React | 16.14.0 |
| System | MacOS Big Sur / Windows 10 |
| Browser | Chrome 94.0.4606.81 / Safari 14.1.2 (only on MacOS) / Firefox 93.0 |

**This is an Accessibility issue. The current solution fails WCAG 2.1 Success Criterion 2.1.3** 

[WCAG 2.1 Success Criterion 2.1.3 - Keyboard (No exception)](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-no-exception)

Our corporate UI development team potentially has capability to implement a fix for this to support our product.

**Comment by afc163** [comment]
fixed in https://github.com/ant-design/ant-design/pull/49333
